app.js

```diff
@@ -24,17 +24,21 @@ app.use(session({
 }));
 
 var PORT = process.env.PORT || 3000;
-var mongoURI = process.env.MONGOURI;
+var mongoURI = process.env.MONGOURI || 'mongodb://localhost/test';
+mongoose.connect(mongoURI);
 
-app.get('/', index.home);
+app.get('/', index.login);
+app.get('/login', index.login);
+app.get('/home', index.home);
 
 app.post('/loginUser', index.loginUser);
 app.post('/addingTwote', index.addTwote);
 app.post('/removingTwote', index.removeTwote);
```

Comment:
There is actually a delete REST call that is in the form app.delete that would perfect here